test/mocha/helpers.js

```diff
@@ -22,11 +24,50 @@ const {Ed25519Signature2018, RsaSignature2018} = suites;
 const sinon = require('sinon');
 const {Ed25519KeyPair} = require('crypto-ld');
 
+const getRandomBytes = promisify(crypto.randomBytes);
+
 // for key generation
 exports.KMS_MODULE = 'ssm-v1';
 // algorithm required for the jwe headers
 exports.JWE_ALG = 'ECDH-ES+A256KW';
 
+const min64Bit = Math.pow(2, 32);
```

Comment:
- This naming here and in test names is confusing.  "min64Bit" seems like it should be 0 or -2^63 (depending if signed).
- JS range is -(2^53 - 1) to 2^53 - 1, so best to avoid saying 64b.

Reply:
would minLarheNumber be better? Any idea what the semantic meaning on the number ranges @dlongley suggested are? What does 2^32 mean? I googled it and it is slightly larger than the upper limitof a 32-bit number so I was assuming this was a 64 vs 32 bit number issue. Perhaps `minBigInt` would be better?

Reply:
Should use `0xFFFFFFFF` and can name it `max32Bit` since that's what it is.

Reply:
This is getting confusing to even talk about. Perhaps code in https://github.com/digitalbazaar/bedrock-edv-storage/pull/32 is easier to understand?

Reply:
With @davidlehn 's changes in are we happy here?